src/st/dre/161DC.c

```diff
@@ -139,7 +139,100 @@ void EntityCSMoveAlucard(Entity* self) {
     }
 }
 
-// Delete this once the below entity is decompiled!
-const s32 rodata_pad_118F0 = 0;
 // appears to load from the CD and freeze the game
-INCLUDE_ASM("st/dre/nonmatchings/161DC", EntityUnkId23);
+void EntityUnkId23(Entity* self) {
+    Entity* player = &PLAYER;
+    Entity* ent = &g_Entities[80];
```

Comment:
I just confirmed in emulator and g_Entities[80] is where the EntitySuccubus is stored, so maybe instead of just "ent" this should be "succubus" or something?

When the Nightmare stage loads, Alucard is forced to walk to the right, outside the player's control; I wonder if this function is responsible for controlling his walking during that loading portion? Not sure.